src/Query.php

```diff
@@ -21,6 +21,9 @@
 
 namespace WikibaseSolutions\CypherDSL;
 
+use Closure;
+use Prophecy\Call\Call;
```

Comment:
I think this shouldn't belong here

Reply:
In case you are interested, I usually run it like `path/to/vendor/bin/php-cs-fixer fix --rules=@PSR12,align_multiline_comment,array_indentation,blank_line_before_statement,indentation_type,no_trailing_whitespace,no_unused_imports,no_useless_else,no_useless_return,ordered_imports,phpdoc_scalar,return_assignment,simplified_if_return,single_line_after_imports,switch_case_space,trailing_comma_in_multiline,ternary_operator_spaces ./src`
 or something along these lines. Fixes most codestyle issues I usually make. Personally I also like the `operator_linebreak` and `single_quote` rules but I won't enforce those on this package.

And there's also a bunch of nice risky fixes that are worth checking out.